lib/static_data/countries.real.json

```diff
@@ -0,0 +1 @@
+../../config/wca-states.json
```

Comment:
The key in the json is `continent_id`, but in the db it's the `continentId` - will this cause any issues with key access?

Reply:
No, because this symlink points to a file maintained by WRC which we parse manually